src/Service/Configurations/RuntimeConfigProvider.cs

```diff
@@ -58,6 +58,11 @@ public virtual string RestPath
             get { return RuntimeConfiguration is not null ? RuntimeConfiguration.RestGlobalSettings.Path : string.Empty; }
         }
 
+        public virtual string RestBaseRoute
+        {
+            get { return RuntimeConfiguration is not null ? RuntimeConfiguration.RestGlobalSettings.BaseRoute : string.Empty; }
+        }
+
```

Comment:
Just noting that #1402 will impact this, primarily because these properties aren't shadowed in the types like this, you always go back to the `RuntimeConfig` type to read it.